components/Navbar.tsx

```diff
@@ -0,0 +1,168 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Container from '@mui/material/Container';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import Tooltip from '@mui/material/Tooltip';
+import MenuItem from '@mui/material/MenuItem';
+
+const pages = ['Book Catalog', 'Members', 'Cafe Schedule', 'Staff'];
+const settings = ['Profile', 'Account', 'Dashboard', 'Logout'];
+
+function Navbar() {
+  const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null);
+  const [anchorElUser, setAnchorElUser] = React.useState<null | HTMLElement>(null);
+
+  const handleOpenNavMenu = (event: React.MouseEvent<HTMLElement>) => {
```

Comment:
I was following the Material UI documentation and it looks like we are using event target to get the specific element that was clicked so that it can be positioned visually instead of just knowing whether the menu is open or not (https://mui.com/material-ui/react-menu/#basic-menu).

Will we be using dropdown menus for some of the button links? Otherwise, I don't think we need it.